mitodl/ocw-hugo-themes
# fix: moving logo from course static to base static

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-hugo-themes/issues/289

#### What's this PR do?
- `Buy at MIT Press` logo was added in `course/static` in this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/336)
 which was merged but it didn't work in RC, the image was not found and threw 404 whereas other static images in `base-theme` and `www` seems to successfully go through.
This logo was also in the build-theme-assets but not sure why it's throwing 404, though it's working locally.
Anyways, I'm opening this PR which moves this logo from `course/static` to `base-theme/static`, just to give it a try as currently, we don't have any static images in `course/static` unlike `base-theme/static`

#### How should this be manually tested?
- Open any course which uses this logo and verify that the logo is rendering and showing up. (e.g: course: `21g.049-spring-2017` uses this logo in [this page](https://ocwnext.odl.mit.edu/courses/21g-049-french-photography-spring-2017/pages/assignments/presentations-and-sources/))

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/93309234/154066969-3c27739e-ae19-4ad7-9de4-afca90591159.png)

![image](https://user-images.githubusercontent.com/93309234/154067006-01601038-0581-47ac-a841-cac5517d61c9.png)
